examples/hand_pose_estimation/hand_keypoints.py

```diff
@@ -0,0 +1,76 @@
+import os
+
+import cv2
+import json
+import numpy as np
+
+from paz.abstract import Loader
+
+
+class HandKeypoints(Loader):
+    def __init__(self, path, split='train'):
+        super().__init__(path, split, None, 'HandKeypoints')
+        self.path = path
+        self.split = split
+
+    def _load_images(self, image_paths):
+        hands = np.zeros((len(image_paths), 300, 300, 3))
+        for arg, image_path in enumerate(image_paths):
+            img = cv2.imread(image_path, cv2.IMREAD_UNCHANGED)
+            resized = cv2.resize(img, (300, 300), interpolation=cv2.INTER_AREA)
+            hands[arg] = resized
+        return hands
+
+    def _load_labels(self, label_paths):
+        keypoints = np.zeros((len(label_paths), 21, 3))
+        for arg, label_path in enumerate(label_paths):
+            with open(label_path, 'r') as fid:
+                data = json.load(fid)
+                keypoint_set = data['hand_pts']
+            keypoints[arg] = keypoint_set.to_numpy().reshape(21, 3)
+        return keypoints
+
+    def _load_paths(self, path):
+        image_paths = list()
+        label_paths = list()
+        if isinstance(self.path, str):
```

Comment:
why is this logic necessary? why don't you just do it in one way only?

Reply:
The CMU dataset used has real-world images and synthetic images in different folders.

I am planning to give the user an option to chose which images they want to use. 

If only manual images, the input from the user is a string but if a combination is needed the input from the user will be a list of strings.

Reply:
I see then we should make this explicit as a new field in the constructor or create two different classes one for synth or one for real